app/src/main/res/layout/gradeitem_node.xml

```diff
@@ -12,31 +12,30 @@
     <TextView
 
         android:id="@+id/txt_assignment_name"
-        android:layout_width="260dp"
+        android:layout_width="240dp"
```

Comment:
@akench Adding `layout_weight` with a non-zero `layout_width` is invalid, try making the width `0dp`. Then the weight should infer the width automatically instead of having to hardcode the width (which prevents the node from being responsive to different device sizes).